qiskit/providers/ibmq/ibmqfactory.py

```diff
@@ -336,7 +352,12 @@ def providers(self, hub=None, group=None, project=None):
 
         return providers
 
-    def get_provider(self, hub=None, group=None, project=None):
+    def get_provider(
+            self,
+            hub: str = None,
+            group: str = None,
+            project: str = None
```

Comment:
same as above, hub/group/project should be `Optional`.

Reply:
Yes, it should. Thanks for the catch Jessie!